virttest/vt_console.py

```diff
@@ -44,6 +52,7 @@ class ConsoleManager(object):
     def __init__(self):
         self._console = None
         self.status_test_command = None
+        self.is_responsive = None
```

Comment:
I assume is_responsive is boolean variable, so self.is_responsive = False??

Reply:
I'd prefer None, which means it's unset. Thanks.